notebooks/en/labelling_feedback_setfit.ipynb

```diff
@@ -0,0 +1,2842 @@
+{
```

Comment:
<p>I think giving a little bit of light context of how you create records in Argilla would be good here? Doesn't need to be super detailed but a high-level overview + link to docs might give people a better sense of what the code they are running is achieving. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>